kiosk/src/Kiosk.css

```diff
@@ -1,98 +1,235 @@
+/* -------- GLOBAL-ISH SYTLING -------- */
+/* ----------------------------------- */
 html, body {
     height: 100%;
     width: 100%;
     padding: 0;
     margin: 0;
     overflow: hidden;
-    background-color: black;
+    background: linear-gradient(rgba(19, 69, 90, 0.81), rgb(5, 5, 5));
     font-family: 'Press Start 2P';
     color: white;
-    align-items: center;
-}
-
-.bg1 {
-    background: radial-gradient(62.83% 55.51% at 50% 50%, rgba(19, 69, 90, 0.81) 0%, #13455A 100%);
-    height: 100%;
-    width: 100%;
-}
-
-.background {
-    box-sizing: border-box;
-    height: 100%;
-    width: 100%;
-    opacity: 0.2;
-    border: 1px solid #141414;
-    filter: blur(137px);
 }
 
 .content {
-    position: absolute;
-    top: 0;
-    left: 0;
-    width: 100%;
+    width: 100vw;
+    height: 100vh;
 }
 
 h1 {
     font-weight: 400;
-    font-size: 54px;
-    line-height: 54px;
+    font-size: 3.375em;
+    line-height: 3.375rem;
     text-align: center;
 }
 
 h2 {
     font-weight: 400;
-    font-size: 44px;
-    line-height: 44px;
+    font-size: 2.75em;
+    line-height: 2.75rem;
     text-align: center;
-    color: white;
 }
 
-.mainMenuHighScoreHeader {
+.sim-embed {
+    width: 100%;
+    height: 100%;
+    top:0;
+    left: -2vw;
+    position: absolute;
+}
+
+.footer {
+    position: fixed;
+    /* These numbers are relative to the size of the footer.png logo */
+    height: 5vh;
+    width: 20vw;
+    left: 1.5%;
+    bottom: 0;
+    margin-bottom: 2vh;
+    background: url("footer.png");
+    background-repeat: no-repeat;
+    background-size: 100%;
+}
+
+/* -------- MAIN MENU SYTLING -------- */
+/* ----------------------------------- */
+
+.mainMenu {
+    width: 100vw;
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+}
+
+.mainMenuHeader {
     font-weight: 400;
-    font-size: 40px;
-    line-height: 40px;
-    text-align: center;
+    font-size: 2.5rem;
     color: white;
 }
 
-.highScoreSmallDiv {
+.mainMenuTopBar {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-around;
+    margin-left: 28.5vw;
+}
+
+.kioskButton {
+    width: 20em;
+    border-color: lightblue;
```

Comment:
always place colors inside of css variables and don't use named colors like this. specify the hex string instead. same for the transparent and lightblue below

this makes it easier for us to do custom themes in the future, like high contrast.